pos_place/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,9 @@
+To configure this module, you need to:
+
+* Add your users to the new group 'Point of Sale - Places' (User or Manager)
+
+* Go to Point of Sale / Configuration / Places
+
+* Create your places
+
+.. figure:: ../static/description/pos_place_tree.png
```

Comment:
I don't know it was possible to specify a path like that, really nice

Reply:
indeed, that the only way to make the images displayed in all cases : 
- on github, in the CONFIGURE.rst view
- in the main README.rst, the path will be correctly generated by oca-gen-readme
- so in the apps store (and OCA store) it is OK
- locally (apps / local modules) it is also OK.